json-schemas/interface-methods/protocol-rule-set.json

```diff
@@ -70,9 +70,32 @@
                 ]
               }
             }
+          },
+          {
+            "required": [
+              "role",
+              "can"
+            ],
+            "properties": {
+              "role": {
+                "$comment": "Must be the protocol path of a record with either $globalRole or $contextRole set to true",
+                "type": "string"
+              },
+              "can": {
+                "type": "string",
+                "enum": [
+                  "read",
+                  "write"
+                ]
+              }
+            }
           }
         ]
       }
+    },
+    "$globalRole": {
```

Comment:
Consideration: would `protocolRole` be better? With complete ignorance the word "global" sounds like something that cuts across protocols. But I also understand push back because one can consider "a context role defined in a protocol config being a type of 'protocol role'"